### DISCUSSION TITLE
Staging URL in configuration file.

### DISCUSSION THREAD

**Discussion by ghost** [braintree/braintree_android#535] [Q&A] [answered]
Hello. I am using your SDK version 2.21.0 and was recently flagged by a security audit for the file orc-config.android.json containing a URL with the word "staging" in it.

File: https://github.com/braintree/braintree_android/blob/5df8408e9ece98a4ceb348c6704940598aa4468f/PayPalOneTouch/otc-config.android.json 

URL: https://assets.staging.braintreepayments.com/one-touch-login/

I am referencing your SDK using the following gradle.build dependency:
implementation('com.braintreepayments.api:braintree:2.21.0') {
    transitive = true
}

Can you tell me if the package received through the above dependency includes this configuration file or URL?

I appreciate your time. Thank you.

Mike

**Comment by hollabaq86** [comment, marked as answer]
please [contact Support](https://developer.paypal.com/braintree/help) for assistance, we do not actively monitor the discussion board for Technical Support.

**Reply by scannillo** [reply]
To add to that - please use our GitHub repo only if you think you've found an issue with our SDK. If you need help troubleshooting your integration, [Braintree Support](https://help.braintreepayments.com) is the place to go. Thanks!

**Comment by scannillo** [comment]
It also looks like you're trying to use v2.21.0 of braintree_android, which is [unsupported](https://github.com/braintree/braintree_android#versions). Please upgrade to the active version (v4) in order to receive troubleshooting help.
